tests/test_schema.py

```diff
@@ -1913,6 +1887,48 @@ class ParentSchema(Schema):
         assert errors == {'foo': {'foo': ['Not a valid integer.']}}
 
 
+class TestPluckSchema:
+
+    @pytest.fixture
+    def user(self):
+        return User(name="Monty", age=81)
+
+    @pytest.fixture
+    def blog(self, user):
+        col1 = User(name="Mick", age=123)
+        col2 = User(name="Keith", age=456)
+        blog = Blog("Monty's blog", user=user, categories=["humor", "violence"],
```

Comment:
Why not write `user=User(name="Monty", age=81)` here rather than create a dedicated single-use `user` fixture?

Reply:
It was copy pasta from copying the setup fixtures from `TestNestedSchema`. Makes sense to me.